src/renderers/webgl/WebGLBindingStates.js

```diff
@@ -16,6 +15,7 @@ function WebGLBindingStates( gl, extensions, attributes, capabilities ) {
 	function setup( object, material, program, geometry, index ) {
 
 		let updateBuffers = false;
+		const vaoAvailable = capabilities.isWebGL2 || extension !== null;
```

Comment:
This line was placed outside of `setup()` on purpose. The idea is to not perform the `vaoAvailable` assessment every time the function is called. 

Besides, WebGL 1 support will be removed next month so this check will be deleted. So there is no need to change it now in any event.

Reply:
Thanks, done!